main.nf

```diff
@@ -1,84 +1,68 @@
 #!/usr/bin/env nextflow
 /*
 ========================================================================================
-                         nf-core/rnaseq
+    nf-core/rnaseq
 ========================================================================================
- nf-core/rnaseq Analysis Pipeline.
- #### Homepage / Documentation
- https://github.com/nf-core/rnaseq
+    Github : https://github.com/nf-core/rnaseq
+    Website: https://nf-co.re/rnaseq
+    lack  : https://nfcore.slack.com/channels/rnaseq
```

Comment:
```suggestion
    Slack  : https://nfcore.slack.com/channels/rnaseq
```

Reply:
👀 Remnant of a late night indent exercise I suspect

Reply:
Fixed https://github.com/nf-core/rnaseq/pull/622/commits/349d083f95a0e995a615424f3f6671ef1e70ebba